contracts/src/v0.8/functions/v1_1_0/libraries/L1Block.sol

```diff
@@ -0,0 +1,122 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.15;
+
+// Copied from the implementation contract for https://optimistic.etherscan.io/address/0x4200000000000000000000000000000000000015#readProxyContract
+// https://optimistic.etherscan.io/address/0x07dbe8500fc591d1852b76fee44d5a05e13097ff#code
+
+import { ISemver } from "./ISemver.sol";
```

Comment:
If we want to do something like this, let's re-use the Chainlink style pattern using `ITypeAndVersion`
```
import {ITypeAndVersion} from "../../../shared/interfaces/ITypeAndVersion.sol";
```